packages/pn-personafisica-webapp/src/component/Notifications/NotificationPayment.tsx

```diff
@@ -274,11 +274,10 @@ const NotificationPayment: React.FC<Props> = ({
     let title = t('detail.payment.summary-pending', { ns: 'notifiche' });
     if (
       paymentInfo.status === PaymentStatus.SUCCEEDED ||
```

Comment:
@leleOFA these two conditions dont need to exist anymore: copy is the same in every case

Reply:
do it